src/content/content.service.spec.ts

```diff
@@ -1,63 +1,233 @@
 import { Test, TestingModule } from '@nestjs/testing';
 import { ContentService } from './content.service';
 import { GitDBService } from '../git-db/git-db.service';
+import { ContentTypeService } from '../content-type/content-type.service';
 
 describe('ContentService', () => {
-  let service: ContentService;
+  let contentService: ContentService;
+  let contentTypeService: ContentTypeService;
   let gitDBService: GitDBService;
 
   beforeEach(async () => {
     const module: TestingModule = await Test.createTestingModule({
-      providers: [ContentService, GitDBService],
+      providers: [ContentService, GitDBService, ContentTypeService],
     }).compile();
 
-    service = module.get<ContentService>(ContentService);
+    contentService = module.get<ContentService>(ContentService);
+    contentTypeService = module.get<ContentTypeService>(ContentTypeService);
     gitDBService = module.get<GitDBService>(GitDBService);
   });
 
   it('should be defined', () => {
-    expect(service).toBeDefined();
+    expect(contentService).toBeDefined();
   });
 
-  it('should call gitDBService getMany', async () => {
-    jest.spyOn(gitDBService, 'list').mockImplementation(() => []);  
-    await service.getMany();
-    expect(gitDBService.getMany).toBeCalled();
+  it('should call gitDb getMany with ids', async () => {
+    const testArray = [1, 2, 3];
+    jest
+      .spyOn(gitDBService, 'getData')
+      .mockImplementation(async () => [1, 2, 3]);
+    const returnedValue = await contentService.getMany('collection', {
+      ids: ['1', '2', '3'],
+    });
+
+    expect(gitDBService.getData).toBeCalledWith(
+      'collection',
+      expect.any(Function),
+    );
+    expect(returnedValue).toStrictEqual(testArray);
+  });
+
+  it('should create correct filter function in getMany', async () => {
+    let testResult;
+    const testArray = [{ id: '1' }, { id: '3' }, { id: '4' }];
+    jest
+      .spyOn(gitDBService, 'getData')
+      .mockImplementation(async (collectionName, func) => {
+        testResult = testArray.filter(func);
+        return [];
+      });
+    await contentService.getMany('collection', { ids: ['1', '4'] });
```

Comment:
actually, we would need to update `ids` to be `id`. `ids` column doesn't exist. 
Let's also update gitdb if needed

Reply:
just rename